src/spider/scheduler/FifoPolicy.cpp

```
@@ -57,89 +60,77 @@ auto task_locality_satisfied(
+    m_metadata_store->get_task_timeout(m_conn, &instances);
+    for (auto const& [instance, task] : instances) {
+        m_tasks.emplace_back(task);
+    }
+
+    // Sort tasks based on job creation time in descending order.
+    // NOLINTNEXTLINE(misc-include-cleaner)
+    absl::flat_hash_map<boost::uuids::uuid, core::JobMetadata, std::hash<boost::uuids::uuid>>
+            job_metadata_map;
+    auto get_task_job_creation_time
+            = [&](boost::uuids::uuid const task_id) -> std::chrono::system_clock::time_point {
+        boost::uuids::uuid job_id;
```

Cppcheck (2.10-2)

[performance] Variable 'm_id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)